product/module.go

```diff
@@ -80,3 +87,16 @@ func (r *routes) Routes(registry *web.RouterRegistry) {
 	h, _ = registry.Route("/product/:marketplacecode/:variantcode/:name.html", `product.view(marketplacecode, variantcode, name, backurl?="")`)
 	h.Normalize("name")
 }
+
+type apiroutes struct {
+	apiController *controller.ApiController
+}
+
+func (r *apiroutes) Inject(apiController *controller.ApiController) {
+	r.apiController = apiController
+}
+
+func (r *apiroutes) Routes(registry *web.RouterRegistry) {
+	registry.Route("/api/v1/products/:marketplacecode", "products.api.get")
```

Comment:
In general I am a fan of API versioning, but the existing API routes do not have it at the moment. So I'm not sure if we should remove it for the sake of consistency.

Reply:
Also why use /product**s**/ and not /product/  like the normal routes?

Reply:
I wanted to add "v1" as additional routes for the existing apis.

Regarding plural: the rest api styles I know prefer plural for collections.
The only other api thats in flamingo-commerce is "cart" and there is no plural since its not a collection but more a "current cart" thing.
What do you think?

Reply:
ah okay had to take a look at REST best practices my self. Plural is fine 👍 
Also saw your other pull requests containing the v1 prefix. thats fine for me as well.